src/support/kuzzle.ts

```diff
@@ -18,7 +18,7 @@ export const kuzzleFlags = {
     default: Boolean(process.env.KUZZLE_SSL) || undefined,
   }),
   username: flags.string({
-    description: 'Kuzzle user',
+    description: 'Kuzzle username (local strategy)',
     default: process.env.KUZZLE_USERNAME || 'anonymous',
```

Comment:
shall 'anonymous' be placed inside a contant.js file to avoid miss typing later on ?

Reply:
IMHO it's look like over engineering to create a new file just to place a default value for the name of Kuzzle anonymous user

Reply:
IMHO This file could be used for all other value and for declaring magic-number also ?